.github/config/out_of_tree_extensions.cmake

```diff
@@ -59,6 +59,15 @@ else ()
     set(LOAD_ICEBERG_TESTS "")
 endif()
 
+################# INET
+duckdb_extension_load(inet
+    DONT_LINK LOAD_TESTS
+    GIT_URL git@github.com:duckdb/duckdb_inet.git
```

Comment:
I would put the plain URL (see iceberg a few lines down), git@github.com I think requires authentication (at least in the default settings), while URL works in a wider range of situations.